src/System.Reflection/src/ApiCompatBaseline.txt

```diff
@@ -0,0 +1,4 @@
+Compat issues with assembly System.Reflection:
```

Comment:
These are CoreCLR issues. If I rename the file to netstandard1.3 at the end, it does not get picked up and the build fails. I see the file without suffix to be used everywhere else to track CoreCLR issues - I do not see why it should be different here.